pkgs/games/purpur/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchurl, nixosTests, jre_headless }:
+let
+  version = "1.17.1";
+  build = "1428";
+in
+stdenv.mkDerivation {
+  pname = "purpur";
+  version = "${version}r${build}";
+
+  src = fetchurl {
+    url = "https://api.purpurmc.org/v2/purpur/${version}/${build}/download";
+    sha256 = "0sjrj67bgkn7ivab36aifds337ps2jajwk9628ilyhlwm1i4l089";
+  };
+
+  preferLocalBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin $out/lib/minecraft
+    cp -v $src $out/lib/minecraft/server.jar
+
+    cat > $out/bin/minecraft-server << EOF
+    #!/bin/sh
+    exec ${jre_headless}/bin/java \$@ -jar $out/lib/minecraft/server.jar nogui
+    EOF
+
+    chmod +x $out/bin/minecraft-server
+  '';
+
+  dontUnpack = true;
+
+  passthru = {
+    tests = { inherit (nixosTests) minecraft-server; };
+  };
+
+  meta = with lib; {
+    description = "Purpur is a drop-in replacement for Paper servers designed for configurability, new fun and exciting gameplay features, and performance built on top of Airplane.";
```

I think this is a bit long for `meta.description`, though it looks good for `meta.longDescription`. Also, it should not have a period at the end.
```suggestion
    description = "A drop-in replacement for Minecraft Paper servers";
    longDescription = ''
      Purpur is a drop-in replacement for Minecraft Paper servers designed for configurability, new fun and exciting 
      gameplay features, and performance built on top of Airplane.
    '';